doc/connectivity/bluetooth/api/mesh/sar_cfg.rst

```diff
@@ -0,0 +1,238 @@
+
+If the TTL field value is ``0``, the initial value of the timer is set to the unicast retransmissions interval step.
+
+The initial value of the SAR Multicast Retransmissions timer is set to the multicast retransmissions interval.
+
+When the lower transport layer receives a segment of a segmented message, it starts an SAR Discard timer.
+The discard timer tells how long the lower transport layer waits before discarding the segmented message the segment belongs to.
+The initial value of the SAR Discard timer is the discard timeout value indicated by the `SAR Discard Timeout`_ state.
+
+SAR Acknowledgment timer holds the time before a Segment Acknowledgment message is sent for a received segment.
+The initial value of the SAR Acknowledgment timer is calculated using the following formula::
+
+[min(SegN + 0.5 , acknowledgment delay increment) * segment reception interval]
+
+Four counters are related to SAR behavior:
+
+* Unicast retransmissions count (see `SAR Unicast Retransmissions Count`_ state and `SAR Unicast Retransmissions Without Progress Count`_ state)
+* Multicast retransmissions count (see `SAR Multicast Retransmissions Count`_ state)
+* Acknowledgment retransmissions count (see `SAR Acknowledgment Retransmissions Count`_ state)
+
+If the number of segments in the transmission is higher than the value of the `SAR Segments Threshold`_ state, Segment Acknowledgment messages are
+retransmitted using the value of the `SAR Acknowledgment Retransmissions Count`_ state.
+
+.. _bt_mesh_sar_cfg_states:
+
+SAR states
+**********
+
+There are two states defined related to segmentation and reassembly:
+
+* SAR Transmitter state
+* SAR Receiver state
+
+The SAR Transmitter state is a composite state that controls the number and timing of transmissions of segmented messages.
+It includes the following states:
+
+* SAR Segment Interval Step
+* SAR Unicast Retransmissions Count
+* SAR Unicast Retransmissions Without Progress Count
+* SAR Unicast Retransmissions Interval Step
+* SAR Unicast Retransmissions Interval Increment
+* SAR Multicast Retransmissions Count
+* SAR Multicast Retransmissions Interval Step
+
+The SAR Receiver state is a composite state that controls the number and timing of Segment Acknowledgment transmissions and the discarding of reassembly of a segmented message.
+It includes the following states:
+
+* SAR Segments Threshold
+* SAR Discard Timeout
+* SAR Acknowledgment Delay Increment
+* SAR Acknowledgment Retransmissions Count
+* SAR Receiver Segment Interval Step
+
+SAR Segment Interval Step
+=========================
+
+SAR Segment Interval Step state holds a value that controls the interval between transmissions of segments of a segmented message.
+Use the :kconfig:option:`CONFIG_BT_MESH_SAR_TX_SEG_INT_STEP` Kconfig option to set a value.
+Segment transmission interval is then calculated using the following formula::
+
+(:kconfig:option:`CONFIG_BT_MESH_SAR_TX_SEG_INT_STEP` + 1) * 10 ms
```

Comment:
you can't use the `:kconfig:option:` role within a code block as it doesn't render

<img width="640" alt="image" src="https://github.com/zephyrproject-rtos/zephyr/assets/128251/11627bd7-f151-4506-aa51-386afbe8748a">

Reply:
Of course, I forgot to think of that. Removed the :kconfig:option:``